wp-content/mu-plugins/postcrafter-yoast-integration/tests/test-data-conversion.php

```
@@ -0,0 +1,910 @@
+        
+        $post_id = $this->test_post_ids[2];
+        
+        // Test 1: Yoast robots to RankMath array
+        $this->run_test(
+            'Yoast Robots to RankMath Array',
+            function() use ($post_id) {
+                // Set up mixed Yoast robots data
+                update_post_meta($post_id, '_yoast_wpseo_meta-robots-noindex', '1');
+                update_post_meta($post_id, '_yoast_wpseo_meta-robots-nofollow', '');
+                
+                $result = $this->converter->convert_between_plugins(
```

PHPMD (2.15.0)

Avoid unused local variables such as '$result'. (Unused Code Rules)

(UnusedLocalVariable)